Library/Homebrew/formula_installer.rb

```diff
@@ -786,7 +791,7 @@ def pour
       path.extend(InstallRenamed)
       path.cp_path_sub(formula.bottle_prefix, HOMEBREW_PREFIX)
     end
-    FileUtils.rm_rf formula.bottle_prefix
+    FileUtils.rm_rf Dir["#{formula.bottle_prefix}/{etc,var}"]
```

Comment:
How about just `FORMULA/VERSION/.formula/FORMULA.rb`? If you're 👍 on that and this PR is updated accordingly think this is good to 🚢. Thanks for your patience here 🙇